R/CohortConstruction.R

```diff
@@ -371,15 +371,17 @@ instantiateCohortSet <- function(connectionDetails = NULL,
                                  results_database_schema.cohort_inclusion_result = "#cohort_inc_result",
                                  results_database_schema.cohort_inclusion_stats = "#cohort_inc_stats",
                                  results_database_schema.cohort_summary_stats = "#cohort_summary_stats",
-                                 warnOnMissingParameters = FALSE)
+                                 warnOnMissingParameters = FALSE,
+                                 episodetable = FALSE)
       } else {
         sql <- SqlRender::render(sql,
                                  cdm_database_schema = cdmDatabaseSchema,
                                  vocabulary_database_schema = cdmDatabaseSchema,
                                  target_database_schema = cohortDatabaseSchema,
                                  target_cohort_table = cohortTable,
                                  target_cohort_id = cohorts$cohortId[i],
-                                 warnOnMissingParameters = FALSE)
+                                 warnOnMissingParameters = FALSE,
+                                 episodetable = FALSE)
```

Comment:
@denyskaduk Do we expect a data source with episode tables to edit this source file? Shouldn't there be a parameter in the main run method?

Reply:
I'll work on the parametrization of the episode tables this week